messages/context.json

```
@@ -1,6 +1,7 @@
 {
   "store/search.placeholder": "search.placeholder",
   "store/search.searchFor": "search.searchFor",
+  "store/search.searchTermTooShort": "search.searchTermTooShort",
```

```suggestion
  "store/search.search-term-too-short": "search.searchTermTooShort",
```

This is a bit weird because most of the handles are in camel case, but we are trying to put them all in kebab case